src/celeritas/phys/PhysicsParams.cc

```diff
@@ -90,6 +89,13 @@ PhysicsParams::PhysicsParams(Input inp)
         inp.action_registry->insert(pre_step_action);
         pre_step_action_ = std::move(pre_step_action);
 
+        auto msc_action = make_shared<ImplicitPhysicsAction>(
+            action_reg.next_id(),
+            "msc-range",
+            "range limitation due to multiple scattering");
```

Comment:
This is a bit of a hack too... but with the current `AlongStepFactory` implementation we can't create new implicit IDs when loading the MSC parameters 🤔 but I think a solution may become apparent down the line.